.circleci/config.yml

```diff
@@ -162,31 +162,78 @@ jobs:
           - ~/.m2
       - <<: *push_images
 
-  # Connector depends on integration, mount workspace .m2
-  connector:
+  # Connector base support depends on integration, mount workspace .m2
+  connector-base:
     <<: *job_defaults
     environment:
       <<: *common_env
     steps:
       - checkout
       - restore_cache:
-          key: syndesis-mvn-connector-{{ checksum "app/connector/pom.xml" }}
+          key: syndesis-mvn-connector-{{ checksum "app/connector-support/pom.xml" }}
       - <<: *load_m2
       - run:
-          name: Build Connectors
+          name: Build Connector Util
+          command: |
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-util | tee -a build_log.txt
+      - run:
+          name: Build Connector Verifier
           command: |
-            ./tools/bin/syndesis build --batch-mode --module connector | tee build_log.txt
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-verifier | tee -a build_log.txt
+      - run:
+          name: Build Connector Maven Plugin
+          command: |
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-maven-plugin | tee -a build_log.txt
+      - run:
+          name: Build Connector Catalog
+          command: |
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-catalog | tee -a build_log.txt
+      - run:
+          name: Build Connector Processor
+          command: |
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-processor | tee -a build_log.txt
+      - run:
+          name: Build Connector Test
+          command: |
+            ./tools/bin/syndesis build --batch-mode --module :connector-support-test | tee -a build_log.txt
       - <<: *save_junit
       - store_test_results:
           path: /workspace/junit
       - store_artifacts:
           path: build_log.txt
       - save_cache:
-          key: syndesis-mvn-connector-{{ checksum "app/connector/pom.xml" }}
+          key: syndesis-mvn-connector-{{ checksum "app/connector-support/pom.xml" }}
           paths:
           - ~/.m2
       - <<: *save_m2
 
+  # Connector parameterized depends on connector-base, mount workspace .m2
+  connector:
```

Comment:
Here `connector` is a job, I think it needs to be a [command](https://circleci.com/docs/2.0/reusing-config/#using-the-parameters-declaration)

Reply:
I wanted to have a parameterized job to call it from the workflow: https://circleci.com/docs/2.0/reusing-config/#authoring-parameterized-jobs

Reply:
Oh cool.